src/tree/ContainerAppItem.ts

```diff
@@ -92,6 +92,10 @@ export class ContainerAppItem implements ContainerAppsItem, RevisionsDraftModel
                 this.containerApp.id,
                 this.containerApp.name);
 
+            if (this.containerApp.provisioningState && this.containerApp.provisioningState !== 'Succeeded') {
```

Comment:
The error makes it sound like the user needs to do something. I assume that they just need to wait, right?

Reply:
It depends, if the provisioning is "InProgress" then they can just wait and try again.  But if the provisioning status is "Failed" they might need to do something

Reply:
Then may be worth telling the user what the state is and give them appropriate responses. Otherwise this error message isn't much more helpful.

Reply:
The provision state is shown in the tree item description, so I kind of assumed the user could figure it out pretty easily on their own, I was just trying to avoid the nonNull throw because that's truly confusing

Reply:
Yeah, I was just thinking we could change the error to something more like:

`The container app "{0}" cannot be expanded due to its provisioning state of "InProgress". Wait for provisioning to complete."

`The container app "{0}" cannot be expanded due to its provisioning state of "Failed". The provision must be rectified to view this resource."